src/particle/pdgid/__init__.py

```diff
@@ -68,13 +68,17 @@
 from .functions import is_valid
 from .functions import abspid
 
+from .functions import is_quark
 from .functions import is_lepton
 from .functions import is_hadron
 from .functions import is_meson
 from .functions import is_baryon
+from .functions import is_heavy_flavor
 from .functions import is_pentaquark
+from .functions import is_gauge_boson_or_higgs
 from .functions import is_sm_gauge_boson_or_higgs
 from .functions import is_generator_specific
+from .functions import is_special_particle
```

Comment:
Is the difference between special particle and generator specific clear?

Reply:
I simply took the classification from the PDG MC numbering scheme document. To be fair, I do not expect most people to ever have to care about these, and generator-specific particles/concepts are the most relevant ones. But, since the definition is not trivial - special set of particles - I thought I should provide it. In the doc I give examples and your comment makes me think that I should rather give the full list of "special particles" taken into account. I will update the doc a bit.

Reply:
I further developed the docstring. It should be very clear now what is meant.